packages/cli/package.json

```diff
@@ -57,7 +57,8 @@
     "regex-parser": "2.2.11",
     "rimraf": "3.0.2",
     "typescript": "4.0.7",
-    "ws": "7.3.1"
+    "ws": "7.3.1",
+    "yaml": "^1.10.2"
```

Comment:
There's no need to add another yaml package, we use the `js-yaml` package in the CLI package, as well as in the rest of the packages in the monorepo that need to serialize / deserialize YAML.